### PR TITLE
Adding KS Technologies Vendor and Impulse (KST5420) Product

### PR DESCRIPTION
**Pull request by Devin-Saavedra**
#### Summary
Adding KS Technologies Vendor and Impulse (KST5420) Product

#### Changes
Adding KS Technologies Vendor and Impulse (KST5420) Product

### CODE REVIEW

File: vendor/ks-technologies/kst5420.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+name: Impulse
```
Comment (LDannijs) [inline vendor/ks-technologies/kst5420.yaml:1]: If possible, could you provide a description of the product in the name too? So something along the lines of `Impulse - Coverage Tool`

File: vendor/index.yaml [hunk 2]
```diff
@@ -2006,3 +2006,13 @@ vendors:
   - id: tip
     name: TIP Meter
     website: https://www.tip-meter.de/
+
+  - id: ks-technologies
```
[the remark this answers is not in the document]
Comment (Devin-Saavedra) [reply vendor/index.yaml]: Hey @Jaime-Trinidad, Sorry about that. Just fixed this in the the latest commit. Do I need to issue another PR?

File: vendor/ks-technologies/kst5420.yaml (hunk 1 above)
Comment (Devin-Saavedra) [reply vendor/ks-technologies/kst5420.yaml]: No problem, just updated the name with a description in the latest commit @LDannijs, do I need to issue another PR?

Comment (LDannijs) [reply vendor/ks-technologies/kst5420.yaml]: No you do not need to. I'll just review it again.

**Review by LDannijs — APPROVED** [review]

**Comment by Jaime-Trinidad** [comment]
@Devin-Saavedra there are still some conflicts to resolve, can you take a look at them?

**Comment by Devin-Saavedra** [comment]
@Jaime-Trinidad Odd that there is a conflict still. I did a fresh fork, added in our updates, and then issued a PR. I will take a look.

File: vendor/index.yaml [hunk 3]
```diff
@@ -1467,8 +1467,14 @@ vendors:
     vendorID: 798
 
   - id: ks-technologies
-    name: KS Technologies, LLC
+    name: KS Technologies
     vendorID: 800
+    website: https://kstechnologies.com/
+    description: Your Vision. Our Solution.
+    social:
+      twitter: https://twitter.com/kstechnologies
```
[the remark this answers is not in the document]
Comment (Devin-Saavedra) [reply vendor/index.yaml]: Hey @LDannijs Good catch, I looked at how others did this and updated ours to just be the twitter handle name vs the full URL. 
I do not know what that conflict is in the vendor/index.yaml file though, as I did not touch any other vendors info in there.

[the remark this answers is not in the document]
Comment (LDannijs) [reply vendor/index.yaml]: Could you run `make validate fmt`? If you get the `EAGAIN` error use `make validate fmt -i`. There's probably still some formatting issues.

[the remark this answers is not in the document]
Comment (Devin-Saavedra) [reply vendor/index.yaml]: This is what I get after running `make validate fmt`

```
❯ make validate fmt
npm run validate

> lorawan-devices@1.0.0 validate
> node bin/validate.js

node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module 'ajv'
Require stack:
- /lorawan-devices/bin/validate.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (/Users/devin/Desktop/Projects/lorawan-devices/bin/validate.js:3:13)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Module._load (node:internal/modules/cjs/loader:1023:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/lorawan-devices/bin/validate.js' ]
}

Node.js v21.1.0
make: *** [validate] Error 1
```

[the remark this answers is not in the document]
Comment (LDannijs) [reply vendor/index.yaml]: Have you made sure to install all the dependencies? https://github.com/TheThingsNetwork/lorawan-devices/pull/726

Also i see you're using Node.js v21. We recommend switching to v16 (with nvm for example)

[the remark this answers is not in the document]
Comment (Devin-Saavedra) [reply vendor/index.yaml]: Sorry about that @LDannijs. I am using nvm now and have the following versions installed:
* node 16.20.0
* npm 8.19.4
* go 1.18.10

After running `make validate fmt` I get this error at the very end:
```
node:events:491
      throw er; // Unhandled 'error' event
      ^

Error: read ENOTCONN
    at tryReadStart (node:net:662:20)
    at Socket._read (node:net:673:5)
    at Socket.Readable.read (node:internal/streams/readable:487:10)
    at Socket.read (node:net:729:39)
    at new Socket (node:net:449:12)
    at Object.Socket (node:net:318:41)
    at createSocket (node:internal/child_process:335:14)
    at ChildProcess.spawn (node:internal/child_process:452:23)
    at spawn (node:child_process:733:9)
    at Object.execFile (node:child_process:331:17)
Emitted 'error' event on Socket instance at:
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -57,
  code: 'ENOTCONN',
  syscall: 'read'
}
make: *** [validate] Error 1
```

I am unsure of how to resolve that `ENOTCONN` error.

[the remark this answers is not in the document]
Comment (LDannijs) [reply vendor/index.yaml]: Hey Devin, have you updated your fork/branch to the current one on the main one? We've made some changes to the validation to hopefully mitigate some errors so.

[the remark this answers is not in the document]
Comment (Devin-Saavedra) [reply vendor/index.yaml]: Hey @LDannijs I have not. Let me do that and try again! Thank you for your help here.

**Pull request closed without merging**